tensorflow/core/ops/ctc_ops.cc

```diff
@@ -62,6 +62,43 @@ REGISTER_OP("CTCLoss")
       return Status::OK();
     });
 
+REGISTER_OP("CTCLossV2")
+    .Input("inputs: float")
```

Comment:
Does the CuDNN implementation support types other than float? If so, we should also support them here.

https://github.com/tensorflow/tensorflow/pull/31164 added support for double for CTCLossOp, for example.

Reply:
No, CuDNN only support the float CTCLoss.